scene/2d/cpu_particles_2d.cpp

```diff
@@ -671,12 +671,15 @@ void CPUParticles2D::_particles_process(float p_delta) {
 			restart = true;
 		}
 
+		float tv;
```

Comment:
Should be initialized to `0` in case there's any branch which doesn't assign it a value below (currently there's none, but it's better to do it now than to get bitten by uninitialized memory in the future).

Reply:
Fixed (in all files)